src/openms/source/FORMAT/MzTab.cpp

```diff
@@ -2394,7 +2394,7 @@ Not sure how to handle these:
       const Feature& f = feature_map[i];
       vector<String> keys;
       f.getKeys(keys); //TODO: why not just return it?
-      replaceWhiteSpaces_(keys.begin(), keys.end());
+      std::transform(keys.begin(), keys.end(), keys.begin(), [&](String& s) { return s.substitute(' ', '_'); });
```

Comment:
```suggestion
      // replace whitespaces with underscore
      std::transform(keys.begin(), keys.end(), keys.begin(), [&](String& s) { return s.substitute(' ', '_'); });
```

Reply:
You use this funcitonality a few times in the cpp file. You could just add it to the cpp file as a member function (not the header)

Reply:
alternatively add a comment to each of them